python/cudf/cudf/core/frame.py

```diff
@@ -238,6 +239,66 @@ def _quantiles(
         result._copy_categories(self)
         return result
 
+    def repeat(self, repeats, axis=None):
+        assert axis in (0, None)
```

Comment:
Would be good to make this assertion more guiding for the user and raise a `NotImplementedError` if axis is 1.